### DISCUSSION TITLE
How to track the position of the window after it is moved

### DISCUSSION THREAD

**Discussion by tatarnikov-yura** [flet-dev/flet#3815] [Q&A] [answered]
### Question

I need to record the coordinates of a form without a frame after moving it using WindowDragArea, how to track the end of moving the form.

**Comment by ndonkoHenri** [comment]
I personally dont seem to understand the question. Can you reformulate?

**Reply by tatarnikov-yura** [reply]
In the Windows application. After moving the window, I need to record the position of the window.
How do I track the release of the mouse button after moving?

**Reply by ndonkoHenri** [reply, marked as answer]
Suggestion: [listen](https://flet.dev/docs/controls/page#on_window_event) to window events, precisely the [`MOVED`](https://flet.dev/docs/reference/types/windoweventtype#moved) event, which occurs after the window was moved. When it occurs read the window's position from [`page.window`](https://flet.dev/docs/controls/page#window), through props like `top`, `left`, `right` and `bottom`.

Let me know if you don't fully understand this suggestion and eventually, if it worked.
